h2o-py/h2o/__init__.py

```diff
@@ -61,12 +61,31 @@ def readTxtFromWhl(name, fallback):
 if (__version__.endswith("99999")):
     print(__buildinfo__)
 
-__all__ = ("connect", "init", "api", "connection", "upload_file", "lazy_import", "import_file", "import_sql_table",
+
+__all__ = ["connect", "init", "api", "connection", "upload_file", "lazy_import", "import_file", "import_sql_table",
            "import_sql_select", "parse_setup", "parse_raw", "assign", "deep_copy", "get_model", "get_grid", "get_frame",
            "show_progress", "no_progress", "enable_expr_optimizations", "is_expr_optimizations_enabled", "log_and_echo",
            "remove", "remove_all", "rapids", "ls", "frame", "import_hive_table",
            "frames", "download_pojo", "download_csv", "download_all_logs", "save_model", "download_model", "upload_model", "load_model", "export_file",
            "cluster_status", "cluster_info", "shutdown", "create_frame", "interaction", "as_list", "network_test",
-           "set_timezone", "get_timezone", "list_timezones", "demo", "make_metrics", "cluster", "load_dataset","flow",
+           "set_timezone", "get_timezone", "list_timezones", "demo", "make_metrics", "cluster", "load_dataset", "flow",
            "upload_custom_metric", "upload_custom_distribution",  "mojo_predict_csv", "mojo_predict_pandas", "import_mojo", 
-           "upload_mojo", "print_mojo", "load_grid", "save_grid")
+           "upload_mojo", "print_mojo", "load_grid", "save_grid"]
+
+try:
+    # Export explain functions that are useful for lists of models
+    from h2o.explain import explain, explain_row, variable_importance_heatmap, model_correlation_heatmap, \
+        partial_dependences, register_explain_methods
+
+    __all__ += [
+        "explain",
+        "explain_row",
+        "variable_importance_heatmap",
+        "model_correlation_heatmap",
+        "partial_dependences"
+    ]
+
+    register_explain_methods()
+except ImportError:
+    pass
```

Comment:
are you using a try-except because `explain` module imports optional dependencies?
If so, ar those optional dependencies used by all methods you want to expose? I mean, if possible, I'd rather do imports  for optional dependencies inside the methods.

Reply:
@sebhrusen Well everything that I expose outside the module uses `matplotlib` and `matplotlib` is optional. I can move it inside the methods if you still think it would be better.

Reply:
if all methods are concerned, then your implementation makes sense.
Just wanted to be sure that we're not preventing users from accessing features that would still work in their env.